devices/schneider_electric.js

```diff
@@ -17,6 +17,21 @@ const tzLocal = {
             return {state: {lift_duration: value}};
         },
     },
+    switchindication: {
+        key: ['switchindication'],
```

Comment:
Please rename this to `indicator_mode` with the following options: `['off', 'off/on', 'on/off', 'on']`

Reply:
Changes made and pushed.